src/reducers/index.js

```diff
@@ -0,0 +1,28 @@
+import { combineReducers, createStore } from 'redux';
+import { books } from './books';
+
+const initialState = [
+  {
+    id: Math.random(),
+    title: 'A Fiction Book',
+    category: 'Sci-Fi',
+  },
+  {
+    id: Math.random(),
+    title: 'Some guy bio',
+    category: 'Biography',
+  },
+  {
+    id: Math.random(),
+    title: 'Watch out, the return',
+    category: 'Horror',
+  },
+];
```

Comment:
`initialState` shoud be inside `src/index.js`
![image](https://user-images.githubusercontent.com/31897434/96794293-d831f500-13f5-11eb-8de7-312f5faf81f3.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/